src/api/mutant/StrykerNode.ts

```diff
@@ -0,0 +1,5 @@
+interface StrykerNode extends ESTree.Node {
+  nodeID: number;
```

Comment:
Well I honestly don't care if we use ID or Id, but I think we should be consistent. Shall I change it to `Id`?